src/Test.FunctionalTests.Sarif/Multitool/BaselineOptionTests.cs

```diff
@@ -81,7 +81,7 @@ private void RunBaselineOptionTest(string testFileName, string testName, bool in
 
         protected override string IntermediateTestFolder => @"Multitool";
 
-        protected override string ConstructTestOutputFromInputResource(string inputResourceName, object parameter)
+        protected override string ConstructTestOutputFromInputResource(string inputResourceName, object parameter, int expectedReturnCode = SUCCESS)
```

Comment:
>, int expectedReturnCode = SUCCESS [](http://example.com/codeflow?start=113&length=34)

because I changed the method signature in base, 
so a few other files in this PR the change is just need to keep the same signature since they override it.